views/js/pciCreator/dev/mathEntryInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -11,6 +11,17 @@
 </div>
 
 <hr />
+<h3>{{__ "Special Settings"}}</h3>
+
+<div>
+    <label class="panel">
+        <input name="spaceBehavesLikeTab" type="checkbox" {{#if spaceBehavesLikeTab}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span>
+        {{__ "prevent white space"}}
```

Comment:
Yes that is right. I confess I just wanted to make this quick.
Ok wait, I will change that.